Process_WOA2018.ipynb

```diff
@@ -4,43 +4,38 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    for period in range(1,2):
</pre><p>12+1 here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/raphaeldussin/remap_WOA18/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>yes. I think I pushed while testing!</p>